test/lib/cql_test_env.cc

```diff
@@ -713,7 +713,9 @@ class single_node_cql_env : public cql_test_env {
             db::view::node_update_backlog b(smp::count, 10ms);
             scheduling_group_key_config sg_conf =
                     make_scheduling_group_key_config<service::storage_proxy_stats::stats>();
-            proxy.start(std::ref(db), std::ref(gossiper), spcfg, std::ref(b), scheduling_group_key_create(sg_conf).get0(), std::ref(feature_service), std::ref(token_metadata), std::ref(erm_factory), std::ref(ms)).get();
+            sharded<service::rpc_fence> rpc_fence;
```

Comment:
Should we call it topology_service in anticipation of more roles?

I think traditionally we'd put it in storage_service, but I'm happy to pull things out into new services, to reduce that tangled mess.